src/components/BorrowModal.tsx

```
@@ -0,0 +1,448 @@
+import { useAccount, useBalance, useSwitchChain } from 'wagmi';
+import Input from '@/components/Input/Input';
+import AccountConnect from '@/components/layout/header/AccountConnect';
+import { useLocalStorage } from '@/hooks/useLocalStorage';
+import { useOraclePrice } from '@/hooks/useOraclePrice';
+import { useStyledToast } from '@/hooks/useStyledToast';
+import { formatBalance, formatReadable } from '@/utils/balance';
+import { findToken } from '@/utils/tokens';
+import { Market } from '@/utils/types';
+import { BorrowProcessModal } from './BorrowProcessModal';
+import { Button } from './common';
+import useUserPosition from '@/hooks/useUserPosition';
```

ESLint

[error] `@/hooks/useUserPosition` import should occur before import of `@/utils/balance`

(import/order)